src/copilot/inspect/InspectionCopilot.ts

```diff
@@ -132,43 +137,53 @@ export default class InspectionCopilot extends Copilot {
     }
 
     public async inspectRange(document: TextDocument, range: Range | Selection): Promise<Inspection[]> {
-        // ajust the range to the minimal container class or (multiple) method symbols
-        const methods: SymbolNode[] = await getIntersectionMethodsOfRange(range, document);
-        const classes: SymbolNode[] = await getClassesContainedInRange(range, document);
-        const symbols: SymbolNode[] = [...classes, ...methods];
-        if (symbols.length < 1) {
-            const containingClass: SymbolNode = await getInnermostClassContainsRange(range, document);
-            symbols.push(containingClass);
+        if (this.busy) {
+            logger.warn('Copilot is busy, please retry after current inspecting tasks is finished.');
+            void window.showWarningMessage(`Copilot is busy, please retry after current inspecting tasks are finished.`);
```

Comment:
When I open more than three Java files, the concurrency exceeds the limit of three. Do you think it’s good to display a message even if the inspection is not manually triggered by the user?